index.js

```diff
@@ -52,7 +52,7 @@ function checkEngines(json, callback) {
       }
 
       done(type, version, range);
-    });
+    }.bind(null, type, range));
```

Comment:
Arguable, but I think it'd be nice if we could avoid using `bind` here. Using a closure could probably achieve the same result. That said, doing so might require some refactoring.

Certainly open to getting a fix in after a rebase and then coming back later to do clean things up.